### ISSUE TITLE
bug: 1.3.4+ the spellcasting system is broken 

### ISSUE THREAD

**Issue by AliDerk** [Sollace/Unicopia#345]
Recipes for crafting most spell types are broken, and as a result, playing as unicorns makes no sense at the moment ![image](https://github.com/Sollace/Unicopia/assets/63415267/8e15f984-8892-4cc0-aff4-7f52f68062af)

**Comment by Sollace** [comment]
It's a known issue. Bug fixes are on my todo list before starting porting to 1.20.5

**Comment by Sollace** [comment]
Duplicate of #342

**Comment by AliDerk** [comment]
I was able to upgrade to a newer version (from 1.20.1 to 1.20.4), and no issues with crafting there
It's worth warning about it somewhere, probably on the same modrinth
